modules/service/src/main/scala/lucuma/odb/graphql/OdbMapping.scala

```diff
@@ -102,11 +100,11 @@ object OdbMapping {
     user0:      User,
     topics0:    Topics[F],
     itcClient:  ItcClient[F],
-    commitHash: CommitHash
-  ):  F[Mapping[F]] =
-    Trace[F].span(s"Creating mapping for ${user0.displayName} (${user0.id}, ${user0.role})") {
-      database.use(enumSchema(_)).map { enums =>
-        new SkunkMapping[F](database, monitor) with BaseMapping[F]
+    commitHash: CommitHash,
+    enums:      Enums
+  ):  Mapping[F] =
+        new SkunkMapping[F](database, monitor)
```

Comment:
Creating the `OdbMapping` no longer requires a trip to the database but I left the indentation for now to avoid obscuring what really changes.